model_analyzer/config/generate/base_model_config_generator.py

```diff
@@ -257,9 +259,17 @@ def make_ensemble_model_config(
         )
 
         model_config_dict["name"] = variant_name
+        logger.info("")
+        if variant_found:
+            logger.info(f"Found existing ensemble model config: {variant_name}")
+        else:
+            logger.info(f"Creating ensemble model config: {variant_name}")
+        for str in logger_str:
```

Comment:
I don't see other changes related to this. Where did logger_str come from?

Reply:
It's passed in to the method. I added this to make ensemble's printout the top-level creation step.